spec/acceptance/class_spec.rb

```diff
@@ -8,7 +8,7 @@
     shell('mkdir -p /etc/puppetlabs/code/environments/production/modules/my_custom_module/files/dashboards')
     context 'default parameters' do
       before do
-        install_module_from_forge('puppetlabs/apt', '>= 7.5.0 < 9.0.0')
+        install_module_from_forge('puppetlabs/apt', '>= 9.1.0 < 10.0.0')
```

Comment:
I don't think we need to change the lower bound here.  That being said, I don't think Puppet check the supported version of puppet and legacy EOL puppet would still try to install 9.x and fail because the code is not compatible, so probably not a big deal :shrug:.

Reply:
yeah that's just for acceptance testing. the lower version doesn't matter.